src/i18n/zh-tw/ui.ts

```diff
@@ -126,6 +128,8 @@ export default UIDictionary({
 	'stub.description.cms': '知道更多在 Astro 使用此 CMS 的資訊？',
 	'stub.description.backend': '知道更多在 Astro 使用此後端服務的資訊？',
 	// Starlight banner
-	'starlight.title': '想要創建自己的文檔嗎？',
+	'starlight.title': '想要創建自己的文件嗎？',
```

Comment:
```suggestion
	'starlight.title': '想要建立自己的文件嗎？',
```

nit: 「建立」可能更貼近臺灣的習慣，供參考～

Reply:
對，用建立比較好